inflection/CMakeLists.txt

```diff
@@ -130,3 +130,23 @@ make generate-coverage-csv  : Generates code coverage as a csv\\n\
 install(TARGETS   inflection                  LIBRARY      COMPONENT inflection_library)
 install(DIRECTORY ${INFLECTION_INCLUDE_ROOT}/ TYPE INCLUDE COMPONENT inflection_headers)
 install(DIRECTORY ${INFLECTION_DATA_ROOT}/    TYPE DATA    COMPONENT inflection_data)
+
+# CPack Configuration for Ubuntu Packaging
+
+set(CPACK_PACKAGE_NAME "unicode-inflection")
+set(CPACK_PACKAGE_VENDOR "Unicode Consortium")
+set(CPACK_PACKAGE_CONTACT "grhoten@gmail.com")
```

Comment:
I think we need to find another contact. I'm not sure that I'm ready to have my email address here.

Reply:
Is that a mandatory field? If we have to have it, we can use the inflection-team@unicode.org one.

Reply:
Agree.